tests/Traits/EntityReferenceRevisionTrait.php

```diff
@@ -54,4 +55,55 @@ protected function getReferenceRevisionField(string $entity_type, string $bundle
     ];
   }
 
+  /**
+   * Creates a field of an entity reference field storage on the bundle.
+   *
+   * @param string $entity_type
+   *   The type of entity the field will be attached to.
+   * @param string $bundle
+   *   The bundle name of the entity the field will be attached to.
+   * @param string $field_name
+   *   The name of the field; if it already exists, a new instance of
+   *   the existing field will be created.
+   * @param string $field_label
+   *   The label of the field.
+   * @param string $target_entity_type
+   *   The type of the referenced entity.
+   * @param string $selection_handler
+   *   The selection handler used by this field.
+   * @param array $selection_handler_settings
+   *   An array of settings supported by the selection handler specified above.
+   *   (e.g. 'target_bundles', 'sort', 'auto_create', etc).
+   * @param int $cardinality
+   *   The cardinality of the field.
+   *
+   * @see \Drupal\Core\Entity\Plugin\EntityReferenceSelection\SelectionBase::buildConfigurationForm()
```

Comment:
SelectionBase is deprecated in drupal:8.0.0 and is removed from drupal:9.0.0. 
Might be we should point DefaultSelection:buildConfigurationForm()?